commons/passive/fitbit/fitbit_heart_rate.avsc

```diff
@@ -0,0 +1,30 @@
+{
+  "namespace": "org.radarcns.passive.fitbit",
+  "type": "record",
+  "name": "FitbitIntradayHeartRate",
+  "doc": "Intra day heart rate data from fitbit device",
+    "fields": [
+        {
+            "name": "type",
+            "type": "string",
+            "doc": "Type of data coming from fitbit intraday API, (e.g. heart rate)"
+        },
+        {
+            "name": "datasetInterval",
```

Comment:
Could datasetInterval and datasetType be merged to a `interval` or `window` field, type `double`, containing `Chronological window size (s).`?